README.md

```diff
@@ -54,7 +54,12 @@
 ### Key Features <a name="key-features"></a>
 
```

Comment:
- [ ] On the 'Categories page' I'm not able to see the uploaded icons, It's showing broken images or nothing. Please make sure to fix this issue and users should be able to see the categories images. (If you're facing issues with uploading images, you can also provide option for users to add image link).

![image](https://github.com/martinkarugaba/budget_app/assets/83803414/9c34f79c-6d3b-4384-8c05-a24a94e36007)